lib/defopt.py

```diff
@@ -368,14 +382,24 @@ def signature(func: Callable):
         parameters=parameters, return_annotation=return_annotation)
 
 
-def _populate_parser(func, parser, parsers, short, strict_kwonly,
+def _populate_parser(func, parser, parsers, short, options, strict_kwonly,
                      show_defaults, show_types, no_negated_flags):
+    if strict_kwonly is _unset:
```

Comment:
I would put this check at the top of the definition of bind(), immediately normalizing to `options` and not bothering to pass `strict_kwonly` down the chain anywhere else.